commit 415c75677544df6aa4929c851f44a412ea39b7b4
Author: Emmanuele Bassi <ebassi@gnome.org>
Date:   Fri Jul 25 09:10:52 2014 +0100

    Include "config.h" in the enum types template
    
    Expands the symbol annotation correctly.
---
 json-glib/json-enum-types.c.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/json-glib/json-enum-types.c.in b/json-glib/json-enum-types.c.in
index 8b82049..0e09075 100644
--- a/json-glib/json-enum-types.c.in
+++ b/json-glib/json-enum-types.c.in
@@ -1,4 +1,5 @@
 /*** BEGIN file-header ***/
+#include "config.h"
 #include "json-enum-types.h"
 /*** END file-header ***/
 
